EcoSoapBank/App/AppFlowCoordinator.swift

```diff
@@ -52,13 +52,22 @@ class AppFlowCoordinator: FlowCoordinator {
 
         // set up window and make visible
         window.rootViewController = tabBarController
-        window.makeKeyAndVisible()
 
         if Keychain.Okta.isLoggedIn {
-            onLoginComplete()
+            userController.logInWithBearer { [weak self] result in
+                switch result {
+                case .failure(let error):
+                    self?.presentLoginFailAlert(error: error)
+                case .success:
+                    self?.onLoginComplete()
+                }
+            }
+            // TODO: show loading screen while logging in
```

Comment:
I realized yesterday we would need get the user (was just using a mock in my PR), but failed to mention it. My apologies for not communicating this.